Code/Editor/TrackView/TrackViewTrack.cpp

```diff
@@ -268,9 +268,9 @@ CTrackViewKeyBundle CTrackViewTrack::GetKeys(const bool bOnlySelected, const flo
         const float keyTime = m_pAnimTrack->GetKeyTime(keyIndex);
         const bool timeRangeOk = (t0 <= keyTime && keyTime <= t1);
 
-        if ((!bOnlySelected || IsKeySelected(keyIndex)) && timeRangeOk)
+        if (timeRangeOk && (!bOnlySelected || IsKeySelected(keyIndex)))
         {
-            CTrackViewKeyHandle keyHandle(this, keyIndex);
+            CTrackViewKeyHandle keyHandle(const_cast<CTrackViewTrack*>(this), keyIndex);
```

Comment:
Are we sure we won't run into undefined behavior because of this in the long run by whomever end ups using the keyHandle ? To me it does not seems worth marking this method as const

Reply:
Maybe worth waiting for further refactoring to use CTrackViewKeyConstHandle as return value instead